### PR TITLE
created and formatted 'about us' page and changed some home page (top…

### PR DESCRIPTION
**Pull request by junseo-lee-git**
… image, button color, navbar size)

### CODE REVIEW

File: react-frontend/src/components/Navbar/Navbar.js
Comment (Westluu) [inline react-frontend/src/components/Navbar/Navbar.js:1]: Nice, makes it more readable

File: react-frontend/src/pages/about.js
Comment (Westluu) [inline react-frontend/src/pages/about.js:1]: styled components nice, looks nice

**Review by luisdavidgarcia — APPROVED** [review]
Looks fantastic, but few conflicts

**Review by Westluu — APPROVED** [review]
Amazing, good job!

**Pull request merged**